--- a/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/deps/install-deps.sh
+++ b/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/deps/install-deps.sh
@@ -36,6 +36,7 @@ pushd $GPERF_DIR
 touch $LOG
 cp configure.orig configure
 cp Makefile.in.orig Makefile.in
+autoreconf -i
 #make clean
 echo "Echo building dependencies..." >> $LOG
 ./configure $CONFIG_OPTIONS --prefix=${DEPS_DIR}/build --enable-frame-pointers --with-pic 2>&1 >> $LOG || echo "Failed in configure for gperftools" >> $LOG
